src/_P103_Atlas_EZO_pH_ORP_EC.ino

```diff
@@ -0,0 +1,641 @@
+                                    0,
+                                    14,
+                                    2,
+                                    0.01);
+
+            if (nb_calibration_points > 0) {
+                addHtml(F("&nbsp;<span style='color:green;'>OK</span>"));
+            } else {
+                addHtml(F("&nbsp;<span style='color:red;'>Not yet calibrated</span>"));
+            }
+            addFormCheckBox(F("Enable"), F("Plugin_103_enable_cal_M"), false);
+            addHtml(F(
+                        "\n<script type='text/javascript'>document.getElementById(\"Plugin_103_enable_cal_M\").onclick = function(){document.getElementById(\"Plugin_103_enable_cal_L\").checked = false;document.getElementById(\"Plugin_103_enable_cal_H\").checked = false;};</script>\n"));
+
+            addRowLabel(F("<strong>Low</strong>"));
+            addFormFloatNumberBox(F("Ref Ph"),
+                                    F("Plugin_103_ref_cal_L"),
+                                    PCONFIG_FLOAT(2),
+                                    0,
+                                    14,
+                                    2,
+                                    0.01);
+
+            if (nb_calibration_points > 1) {
+                addHtml(F("&nbsp;<span style='color:green;'>OK</span>"));
+            } else {
+                addHtml(F("&nbsp;<span style='color:red;'>Not yet calibrated</span>"));
+            }
+            addFormCheckBox(F("Enable"), F("Plugin_103_enable_cal_L"), false);
+            addHtml(F(
+                        "\n<script type='text/javascript'>document.getElementById(\"Plugin_103_enable_cal_L\").onclick = function(){document.getElementById(\"Plugin_103_enable_cal_M\").checked = false;document.getElementById(\"Plugin_103_enable_cal_H\").checked = false;};</script>\n"));
+
+            addHtml(F("<TR><TD><strong>High</strong></TD>"));
+            addFormFloatNumberBox(F("Ref Ph"),
+                                    F("Plugin_103_ref_cal_H"),
+                                    PCONFIG_FLOAT(3),
+                                    0,
+                                    14,
+                                    2,
+                                    0.01);
+
+            if (nb_calibration_points > 2) {
+                addHtml(F("&nbsp;<span style='color:green;'>OK</span>"));
+            } else {
+                addHtml(F("&nbsp;<span style='color:orange;'>Not yet calibrated</span>"));
+            }
+            addFormCheckBox(F("Enable"), F("Plugin_103_enable_cal_H"), false);
+            addHtml(F(
+                        "\n<script type='text/javascript'>document.getElementById(\"Plugin_103_enable_cal_H\").onclick = function(){document.getElementById(\"Plugin_103_enable_cal_L\").checked = false;document.getElementById(\"Plugin_103_enable_cal_M\").checked = false;};</script>\n"));
+
+            if (nb_calibration_points > 1) {
+                char sensordata[32];
```

Comment:
There is already an object in this scope with the same name and signature. Is that correct?

Reply:
yes. I will put this in a seperate function

Reply:
now in separate function